example/spec/config.yml

```diff
@@ -1,6 +1,7 @@
 stackName: test-npm-integration
 prefix: test-npm-integration
 bucket: cumulus-test-sandbox-internal
+distributionEndpoint: https://3tu0izwxuc.execute-api.us-east-1.amazonaws.com/dev/
```

Comment:
Since we are not actually calling the distribution endpoint to get the files (this is impossible because of sandbox firewalls), let's set the distribution to a dummy url. To do this, we have to carry out the followings:

- add this entry to app/config.yml under default: `api_distribution_url: 'https://example.com`
- change the value here to `https://example.com`